common/scaffolding/typography.html

```diff
@@ -0,0 +1,39 @@
+---
+{
+	"title": "Typography",
+	"language": "en",
+	"altLangPage": "typographie.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-08-24"
+}
+---
+
+<h1>Headings</h1>
+<h1>This is an h1 heading element</h1>
+<h2>This is an h2 heading element</h2>
+<h3>This is an h3 heading element</h3>
+<h4>This is an h4 heading element</h4>
+<h5>This is an h5 heading element</h5>
+<h6>This is an h6 heading element</h6>
+<pre><code>
+&lt;h1&gt;This is an h1 heading element &lt;/h1&gt;
+&lt;h2&gt;This is an h2 heading element &lt;/h2&gt;
+&lt;h3&gt;This is an h3 heading element &lt;/h3&gt;
+&lt;h4&gt;This is an h4 heading element &lt;/h4&gt;
+&lt;h5&gt;This is an h5 heading element &lt;/h5&gt;
+&lt;h6&gt;This is an h6 heading element &lt;/h6&gt;
+</code></pre>
+
+<h1>Paragraph</h1>
```

Comment:
Except for a few exception, a page should only contain one heading level 1

Reply:
Fixed and addressed in commit "Updated headings and added paragraph".